cmd/node/main.go

```diff
@@ -1244,12 +1249,18 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
 		tpsBenchmark,
 		historyRepository,
 		epochNotifier,
+		txSimulatorProcessorArgs,
 	)
 	processComponents, err := factory.ProcessComponentsFactory(processArgs)
 	if err != nil {
 		return err
 	}
 
+	transactionSimulator, err := txsimulator.New(*txSimulatorProcessorArgs)
```

Comment:
New what ? make a better naming for the constructor.

Reply:
ok. renamed